geomstats/_backend/pytorch/__init__.py

```diff
@@ -664,11 +666,22 @@ def array_from_sparse(indices, data, target_shape):
     a : array, shape=target_shape
         Array of zeros with specified values assigned to specified indices.
     """
-    return _torch.sparse.FloatTensor(
-        _torch.LongTensor(indices).t(),
-        _torch.FloatTensor(cast(data, float32)),
-        _torch.Size(target_shape),
-    ).to_dense()
+    if (
```

Comment:
I don't think we should create special cases.

What about?

```
a = _torch.sparse.FloatTensor(
    _torch.LongTensor(indices).t(),
    array(data),
    _torch.Size(target_shape),
).to_dense()
```

Reply:
Ok, thank you!